Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -488,7 +489,14 @@ class TabTrayController: UIViewController, UITabBarDelegate, UICollectionViewDel
             } else {
                 cell.favicon.image = UIImage(named: "defaultFavicon")
             }
-            cell.background.image = tab.screenshot
+
+            if let img = tab.screenshot {
+                cell.background.image = img
+            } else if let thumbs = profile.thumbnails as? SDWebThumbnails {
+                let key = SDWebThumbnails.getKey(tab.url!)
+                cell.background.moz_getImageFromCache(key, cache: thumbs.cache, completed: { img, err, type, key in
+                })
+            }
```

Comment:
Argh. Why did we do that? We already pay for storage for these. Now we're storing them twice?